string-manipulation-1/src/get-words.js

```diff
@@ -0,0 +1,10 @@
+/* exported getWords */
+function getWords(string) {
+  var String;
```

Comment:
💀 💀 💀 💀 💀 

Avoid making variables that start with capital letters. `String` has special meaning in JavaScript, and **_you just blew it up_**. How about `words`?